src/main/java/com/puppycrawl/tools/checkstyle/site/PropertiesMacro.java

```diff
@@ -158,13 +162,34 @@ private static void writeTablePropertiesRows(Sink sink)
         final Map<String, DetailNode> propertiesJavadocs = SiteUtil
                 .getPropertiesJavadocs(properties, currentModuleName, currentModuleFile);
 
-        for (String property : properties) {
+        final Set<String> validProperties = orderProperties(properties);
+
+        for (String property : validProperties) {
             final DetailNode propertyJavadoc = propertiesJavadocs.get(property);
             final DetailNode currentModuleJavadoc = propertiesJavadocs.get(currentModuleName);
             writePropertyRow(sink, property, propertyJavadoc, instance, currentModuleJavadoc);
         }
     }
 
+    /**
+     * Rearrange properties alphabetically with property 'tokens' always last.
```

Comment:
This method does not rearrange anything alphabetically, please do `Reorder properties to always have the 'tokens' property last (if present).`

Reply:
@nrmancuso Changed in new PR